ngrok/src/config/http.rs

```diff
@@ -66,6 +67,33 @@ impl FromStr for Scheme {
     }
 }
 
+/// Restrictions placed on the origin of incoming connections to the edge.
+#[derive(Clone, Default)]
+pub(crate) struct UaFilter {
+    /// Rejects connections that do not match the given CIDRs
+    pub(crate) allow: Vec<String>,
+    /// Rejects connections that match the given CIDRs and allows all other CIDRs.
```

Comment:
Noticed these comments were never switched from CIDR to UA regexes

Reply:
I just never merged the PR to change it :sweat_smile: